src/server/schema.ts

```diff
@@ -63,6 +66,20 @@ export const schema = `
       timeoutMs: Int = 10000
     ): [LookupQueryResult]
   }
+  
+  """
+  The mode in which the literal search query (\`query\`) is interpreted before it is sent to the term sources.
+  """
+  enum QueryMode {
+    "This mode is only for backwards-compatibility. Switch to \`SMART\` mode for forwards-compatbility."
+    DEPRECATED @deprecated(reason: "This mode will be removed in the future. Use \`SMART\` instead.")
+    
+    "Connect search query words with a boolean AND operator and optimise query input for term sources."
+    SMART
+    
+    "Send the unaltered query input to the term sources. For advanced users that want to have full control over the value of the SPARQL parameter."
```

Comment:
Or perhaps not mention SPARQL? For example: in case of Virtuoso the `query` is not processed by the triplestore but send to Virtuoso's underlying full text search engine. Something like '...that want to have full control over the way a term source searches'?

Reply:
Good point; removed the reference to SPARQL.